### ISSUE TITLE
Keyboard accessibility

### ISSUE THREAD

**Issue by rsinghal** [ProjectMirador/mirador#17]
From William Straub/HMML:

For example, according to WCAG standards, users should be able to utilize the keyboard (usually Tab key and arrow keys) to get to any of the menu items – mouse only web buttons and gadgets can be difficult or impossible for some users with disabilities to access

**Comment by mejackreed** [comment]
There is some good information here, but I think the project has moved on with Mirador 3. Closing this issue now.
